src/main/resources/js/all-metrics.js

```diff
@@ -0,0 +1,3 @@
+({width: Math.max(window.innerWidth,document.body.scrollWidth,document.documentElement.scrollWidth)|0,
+height: Math.max(window.innerHeight,document.body.scrollHeight,document.documentElement.scrollHeight)|0,
+deviceScaleFactor: window.devicePixelRatio || 1,mobile: typeof window.orientation !== 'undefined'})
```

Comment:
Add newline at the end of file

Reply:
New line added, but here in PR it is not showed:
![image](https://user-images.githubusercontent.com/39313287/42732654-ebe24dbc-882d-11e8-941c-c0beb1544edb.png)